python/lsst/ts/m2gui/controltab/tab_cell_status.py

```diff
@@ -242,6 +245,51 @@ def _update_figures(self, values, is_displacement):
             else:
                 self._figures[figure_type].axis_y.setTitleText("Force (N)")
 
+    @asyncSlot()
+    async def _callback_time_out(self, threshold=50):
+        """Callback timeout function to update the forces on cell status.
+
```

Comment:
Callback timeout function to update cell status forces (displays?)

Reply:
Updated in the commit: [24c98f6](https://github.com/lsst-ts/ts_m2gui/pull/22/commits/24c98f6e555c1f5d264f9e6b7db25ba443faf54b). Thanks!